test/chen_unit_tests.jl

```diff
@@ -0,0 +1,122 @@
+import Test as TT
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import CLIMAParameters as CP
+import SpecialFunctions as SF
+
+const CMP = CM.Parameters
+const CMC = CM.Common
+const CMT = CM.CommonTypes
+const CMNe = CM.MicrophysicsNonEq
+const CM0 = CM.Microphysics0M
+const CM1 = CM.Microphysics1M
+const CM2 = CM.Microphysics2M
+const CH = CM.TerminalVelocityChen2022
+
+include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))
+
+const Ch2022 = CMT.Chen2022Type()
+
+@info "Chen Terminal Velocity Tests"
+
+function test_chen(FT)
+
+    toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+    prs = cloud_microphysics_parameters(toml_dict)
+
+    TT.@testset "Chen 2022 rain terminal velocity individual" begin
+        #setup
+        ρ = FT(1.1)
+        q_rai = FT(1e-6)
+        N_rai = FT(1e4)
+        D_r = FT(2e-4)
+
+        ϕ = 1
+        q = exp(0.115231 * ρ)
+        ai = [0.044612 * q, -0.263166 * q, 4.7178 * q * ρ^(-0.47335)]
```

Comment:
Thank you for adding the tests! Maybe instead of coding the same thing twice (in the source code and tests) we could instead take the values from the Chen2022 paper. For example for individual rain drop, you could take the values from their figure 3 (you can use tools like https://automeris.io/WebPlotDigitizer/ to read the data from the plot.

For the bulk velocity, I think it's ok to just take some values from our output (instead of computing them again in the test) and compare against them. This is not a test that checks for correctness, but only that things won't change in the future for example when the API changes. Also, if you do that you won't need special functions in your test anymore.

Reply:
Hi! I was thinking of deleting the tests for the individual raindrop since we're not adding that function into the code and only using it for plotting comparisons (let me know if you think that's ok). I will change the bulk velocity though!